templates/batchConversion.tpl

```diff
@@ -0,0 +1,52 @@
+{**
+ * templates/batchConversion.tpl
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2003-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * markup plugin -- displays the ArticlesGrid.
+ *}
+<div>
+	<div class="pkp_form">
+		<fieldset>
+			<legend>Selected submissions</legend>
+			<label class="description">{translate key="plugins.generic.markup.batch.selected-submissions"}</label>
+			<input type="hidden" id="batchFilesToConvert" name="batchFilesToConvert" value="{$batchFilesToConvert}" />
+			<input type="hidden" id="conversionTriggerUrl" name="conversionTriggerUrl" value="{$conversionTriggerUrl}" />
+			<ul id="submissionListConfirmation">
+			{foreach from=$submissions item=submission}
+				<li class="batch-conversion-submission">
+					##{$submission.id} 
```

Comment:
`##` isn't a convention we use elsewhere; just `#` would make more sense, I think.

Reply:
Done.